build-and-deploy.sh

```diff
@@ -6,9 +6,10 @@
 # inspirehep is free software; you can redistribute it and/or modify it under
 # the terms of the MIT License; see LICENSE file for more details.
 
-IMAGE="inspirehep/ui"
-DOCKER_CONTEXT='ui'
-TAG="$(git describe --always)"
+TAG="${TRAVIS_TAG:-$(git describe --always)}"
```

Comment:
```suggestion
TAG="${TRAVIS_TAG:-$(git describe --always --tags)}"
```
in order to allow lightweight (non-annotated) tags too, which are the type of tags generated by github releases.

Reply:
👍 TRAVIS_TAG will be set though